src/components/fxRunner/outputHandler.js

```diff
@@ -98,10 +98,12 @@ module.exports = class OutputHandler {
 
         //Handle script traces
         if(channel == 'citizen-server-impl' && data.type == 'script_structured_trace'){
-            // dir(data.payload)
+			// dir(data.payload)
             if(data.payload.type === 'txAdminHeartBeat'){
                 globals.monitor.handleHeartBeat('fd3');
-            }
+            } else if (data.payload.type === 'txAdminLogData') {
+				globals.databus.serverLog = globals.databus.serverLog.concat(data.payload.logs)
+			}
```

Comment:
fix mixed whitespaces

Reply:
Also in general, make sure you are only using 4 spaces instead of tabs.  
So I guess also fix the other files.